packages/flame/lib/src/text.dart

```diff
@@ -13,6 +13,11 @@ import 'memory_cache.dart';
 ///
 /// See [TextPaint] for the default implementation offered by Flame
 abstract class TextRenderer<T extends BaseTextConfig> {
+  static Map<Type, TextRenderer Function()> defaultCreatorsRegistry = {
+    TextRenderer: () => TextPaint(),
+    TextPaint: () => TextPaint(),
+  };
+
```

Comment:
I wonder if we should just expose the map, or make it private and have a `registerDefaultCreator` or something like that.

I think I prefer the method, but no strong opinions, I leave it up to you.

Reply:
Also, whichever solution we follow, we need to quote this on the docs.

Reply:
I didn't see why make it private, the user can have full control if they want to, for instance, change one of the existing defaults. Just give the power. Also, modifying this at all is a very, very niche thing, so I didn't want to include this since it's already documented on the code.

Reply:
I don't see how this documented on the code, there is not dartdoc on it, or am I missing something?

Reply:
I meant this

![image](https://user-images.githubusercontent.com/882703/120051663-81297980-bfef-11eb-9c44-dd8e509afa53.png)

But I can add a dartdoc as well